romeo/asset-contract/contracts/asset.clar

```diff
@@ -72,6 +77,7 @@
     (begin
         (try! (is-contract-owner))
         (try! (verify-txid-exists-on-burn-chain withdraw-txid burn-chain-height merkle-proof tx-index tree-depth block-header))
+        (asserts! (map-insert amounts-by-btc-tx withdraw-txid (* -1 (to-int amount))) err-btc-tx-already-used)
```

Comment:
I've seen lately that folks are replacing asserts! with assert-eq. Not sure if this is one of those cases.

Reply:
We're doing this (using our custom (assert-eq ...) strictly in the *test* contracts only, not actual Clarity contracts.